tool/tctl/common/auth_command.go

```diff
@@ -276,6 +276,12 @@ func (a *AuthCommand) generateSnowflakeKey(ctx context.Context, clusterAPI auth.
 
 // RotateCertAuthority starts or restarts certificate authority rotation process
 func (a *AuthCommand) RotateCertAuthority(ctx context.Context, client auth.ClientI) error {
+	if a.rotateType == "" {
+		return trace.BadParameter("required flag --type not provided; previous versions defaulted to --type=all which is deprecated and will be removed in a future version")
+	}
+	if a.rotateType == "all" {
+		log.Warningln("--type=all will be deprecated in a future version")
```

Comment:
Can you include a snippet in the PR description of how this appears to the user? Just want to make sure it's obvious and clean without any weird logging artifacts.